lib/pbench/cli/server/tree_manage.py

```diff
@@ -88,18 +109,33 @@ def tree_manage(
         search: Discover cache with a full disk search
     """
     logger = None
+
+    global detailer, verifier, watcher
+    detailer = Detail(detail, False)
```

Comment:
It would be nicer to pass the second parameter by keyword:
```python
    detailer = Detail(detail, errors=False)
```

Reply:
I considered that for half a second and decided not to, because I hadn't done it in the other usage. Still, with the change from an option variable name to a literal we do lose some context and as I'm making other changes I might as well do this.